features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/MediationPolicies/Edit/EditFaultMediationPolicy.jsx

```diff
@@ -250,13 +212,49 @@ function EditFaultMediationPolicy(props) {
                 }
             });
     }
-    const handleDownload = () => {
+    /**
+    * Handles the custom mediation policy delete.
+    * @param {any} policyToDelete policy file id that is to be deleted.
+    */
+    function deleteCustomMediationPolicy(policyToDelete) {
+        const promisedGetContent = API.deleteMediationPolicy(policyToDelete, api.id);
+        promisedGetContent
+            .then(() => {
+                setFaultSeqCustom(faultSeqCustom.filter(seq => seq.id !== policyToDelete));
+                Alert.info(<FormattedMessage
+                    id='Apis.Details.MediationPolicies.Edit.EditInMediationPolicy.delete.success'
```

Comment:
Id needs to be changed as `Apis.Details.MediationPolicies.Edit.EditFaultMediationPolicy.delete.success`

Reply:
Fixed with ff15600df9442db39832474f05d6294dad0c0f50
Thanks.